pom.xml

```diff
@@ -927,19 +927,8 @@
     <profiles>
         <profile>
             <id>ossrh</id>
-            <properties>
```

Comment:
Is it better to have these things in the settings.xml file instead of this profile?